util/dvsim/Deploy.py

```diff
@@ -329,7 +329,7 @@ def dispatch_items(items):
         # Dispatch the given items
         dispatch_items_queue = []
         if len(items) > Deploy.max_parallel:
-            dispatch_items(items[0:Deploy.max_parallel - 1])
+            dispatch_items(items[0:Deploy.max_parallel])
```

Comment:
this bug led to targets that were not built when running with a restricted amount of parallel processes (like 1 or 2).